website/dbt-versions.js

```diff
@@ -26,6 +26,10 @@ exports.versions = [
 ]
 
 exports.versionedPages = [
+  {
+    "page": "reference/resource-properties/unit-tests",
+    "firstVersion": "1.7",
```

Comment:
First dbt-core version will be 1.8

Reply:
Removed the versioning as this will hit Cloud first